formatting_scripts/codespell_ignored_words.txt

```diff
@@ -0,0 +1 @@
+vertexes
```

Comment:
Curious why you needed to add this?

Reply:
One of the pyqtgraph functions has a named parameter `vertexes` which codespell tries to autocorrect to `vertices`

Reply:
Interesting. Does it automatically change the text?

Reply:
Yeah whenever the formatting script is run